live-examples/js-examples/array/array-findlast.js

```diff
@@ -0,0 +1,6 @@
+const array1 = [5, 12, 50, 130, 44];
+
+const found = array1.findLast(element => element > 45);
```

Comment:
Nit. A matter of taste, but also to be coherent with the rest of this PR.
```suggestion
const found = array1.findLast((element) => element > 45);
```

Reply:
THanks!